pkg/steps/cluster_claim.go

```diff
@@ -99,71 +125,59 @@ func acquireCluster(ctx context.Context, clusterClaim api.ClusterClaim, hiveClie
 	}
 
 	clusterPool := clusterPools.Items[0]
-	claimName := jobSpec.ProwJobID
+	claimName := s.jobSpec.ProwJobID
 	claimNamespace := clusterPool.Namespace
-	claim := &hivev1.ClusterClaim{
+	ret := &hivev1.ClusterClaim{
```

Comment:
`ret` is a bad name as it's not descriptive. Why not `claim`?

Reply:
We keep the created one to return because it will be deleted no matter what happens in the acquire function.
Renamed the var `into` which is passed to the acquire function